python/ray/state.py

```diff
@@ -388,6 +388,21 @@ def profile_table(self):
 
         return dict(result)
 
+    def get_placement_group_by_name(self, placement_group_name):
```

Comment:
Can you make this method return the metadata of the placement group entry of the given name instead of just returning an id? (Like same output as `placement_group_table` except that it only returns a single entry). Let's parse the ID in the placement_group.py

Reply:
Yeah, I will update this.